src/features/Dev/DevManager.ts

```diff
@@ -0,0 +1,57 @@
+import moment from "moment";
+
+import { ENTITIES } from "../../common/constants";
+import { DatabaseService } from "../../common/services/Database";
+import { DevService } from '../../common/services/DevService';
+import { Sensor, TemperatureBreachConfiguration, TemperatureLog } from "../../common/services/Database/entities";
+import { UtilService } from "../../common/services/UtilService";
+
+const DELETE_BREACH_LOGS = `
+  DELETE FROM temperaturelog
+  WHERE temperatureLog.timestamp <= ? AND temperatureLog.timestamp >= ?
+`
+
+class DevManager {
+  databaseService: DatabaseService;
+  utilService: UtilService;
+  devService: DevService;
+
+  constructor(databaseService: DatabaseService, utilService: UtilService, devService: DevService) {
+    this.databaseService = databaseService;
+    this.utilService = utilService;
+    this.devService = devService;
+  }
+
+  generateBreachLogs = async (sensor: Sensor) => {
```

Comment:
This guy took me a while, but was worth it to understand how breaches work!

Basic logic is to grab an arbitrary breach config and generate sufficient breach logs to create a temperature breach.